React/wbooks/src/Navbar/Components/Notifications/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import BookCover from '../../../Components/BookCover';
+
+import './style.css';
+
+function Notifications() {
+  return (
+    <div className="notifications-container">
+      <div className="notification">
+        <BookCover imageUrl={null} imgStyle="book-img" defaultStyle="default-img" />
```

Comment:
Why aren't you passing the url?

Even so, don't pass a null imageUrl. Avoid passing the prop altogether instead.